website/content/en/preview/tasks/pod-density.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control Pod Density"
+linkTitle: "Control Pod Density"
+weight: 20
+---
+
+Pod density is the number of pods per node. 
+
+Kubernetes has a default limit of 110 pods per node. Karpener shares this default. 
+
+## Max Pods
+
+Do not use the `max-pods` argument to kubelet. Karpenter is not aware of this value. For example, Karpenter may provision an instance expecting it to accommodate more pods than this static limit. 
+
+## Increase Pod Density
+
+### Networking Limitations 
+
+*☁️ AWS Specific*
+
+The number of pods on a node is limited by the number of networking interfaces (ENIs) that may be attached to a node. 
+
+AWS VPC CNI v1.9 introduced prefix assignment. In short, a single ENI can provide IP addresses for multiple pods. Much higher pod densities are now supported. 
+
+Run the Karpenter controller with the enviornment variable `AWS_ENI_LIMITED_POD_DENSITY` (or the argument  `--aws-eni-limited-pod-density=true`) to enable nodes with more than 110 pods. 
+
+Environment variables for the Karpenter controller may be specified as [helm chart values](https://github.com/aws/karpenter/blob/c73f425e924bb64c3f898f30ca5035a1d8591183/charts/karpenter/values.yaml#L15). 
+
+## Limit Pod Density
```

Comment:
I'm honestly not sure if this section is needed, as it feels more like scheduling concerns (which we already have a doc for).

Reply:
I'm perceiving "fine tune min/max pods per node" and "general scheduling" as different tasks. I'm still thinking though. 

I don't want to bring ENI business into scheduling.